aspnetcore/fundamentals/logging/index.md

```diff
@@ -185,7 +185,7 @@ setx Logging__Console__LogLevel__Microsoft.Hosting.Lifetime Trace /M
 ```
 
 > [!NOTE]
-> When configuring environment variables with names that contain `.` (periods), consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
+> In Linux environments, when configuring configuring environment variables with names that contain `.` (periods), consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
```

Comment:
```suggestion
> When configuring environment variables with names that contain `.` (periods) in macOS and Linux, consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
```

Reply:
With a suggestion I can see the difference and one click accept the suggestion - or all the suggestions I accept.

<img width="327" alt="image" src="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/2b311ede-ee29-4ec4-9b41-7112c8aee5c4">